kuma/static/js/highlight.js

```diff
@@ -0,0 +1,16 @@
+(function(win, doc, $) {
+    'use strict';
+
+    // find all targets and wrap appropriately
+    function highlight(targets) {
+        var $targets = $(targets);
+        $targets.each(function() {
+            $(this).addClass('highlight-spanned').wrapInner('<span class="highlight-span"></span>');
+        });
+    }
+
+    // call on aritcle body with h3 and h4 as targets
+    var $articleSubHeads = $('#wikiArticle h3, #wikiArticle h4');
+    highlight($articleSubHeads);
+
+})(window, document, jQuery);
```

Comment:
I am assuming you chose the JS route because adding a class to all existing `h3` and `h4` tags site wide is going to be next to impossible? So, in this regard it is a progressive enhancement i.e. no-JS === no-Highlight